README.md

```diff
@@ -8,4 +8,18 @@ Also published on GitHub pages: https://catbytes-community.github.io/webplatform
 
 ### Documentation for developers
 
-Located in the `/docs/how-tos` directory. All important information on local run, deployments, etc is stored there.
\ No newline at end of file
+Located in the `/docs/how-tos` directory. All important information on local run, deployments, etc is stored there.
+
+### How to run project locally
+
+1. Follow `/docs/how-tos/aws_local_db_connection` to install AWS CLI and configure AWS credentials, don't have to connect to AWS RDS database from local machine
```

Comment:
AWS access is only needed for remote RDS access. So it's either configuring AWS credentials OR ENVIRONMENT=local + local psql instance